public/app/dashboard/profileController.js

```diff
@@ -27,7 +27,19 @@
 
   'use strict';
 
-  angular.module('collabosphere').controller('ProfileController', function(analyticsService, assetLibraryFactory, me, profileFactory, crossToolRequest, userFactory, utilService, $scope, $state, $stateParams) {
+  angular.module('collabosphere').controller('ProfileController', function(
+    analyticsService,
+    assetLibraryFactory,
+    collaborationMessageService,
+    crossToolRequest,
+    me,
+    profileFactory,
+    userFactory,
+    utilService,
+    $scope,
+    $state,
+    $stateParams
+  ) {
```

Comment:
The change above is [already on master](https://github.com/ets-berkeley-edu/suitec/blob/master/public/app/dashboard/profileController.js#L30). Why the diff suggesting it's not?!